pluto_build/sql/qaqc_outlier.sql

```diff
@@ -0,0 +1,38 @@
+-- -v VERSION=$VERSION
+DROP TABLE IF EXISTS qaqc_outlier;
+CREATE TABLE IF NOT EXISTS qaqc_outlier(
+    v character varying,
+    outlier character varying
+);
+
+DELETE FROM qaqc_outlier
+WHERE v = :'VERSION';
+
+INSERT INTO qaqc_outlier (
+select :'VERSION' as v, 	    
+        jsonb_agg(t) as outlier
+from (
+	select jsonb_agg(json_build_object('bbl', a.bbl, 'unitsres',a.unitsres, 'resarea', a.resarea,
+                              'res_unit_ratio', a.res_unit_ratio)) as values, 'unitsres_resarea' as field
+	from (SELECT DISTINCT bbl, unitsres, resarea, resarea::INT/unitsres::INT as res_unit_ratio
+          FROM archive_pluto
+          WHERE unitsres::INT>50 and resarea::INT/unitsres::INT < 300
+          AND resarea::INT != 0) a
+	union
+	select jsonb_agg(json_build_object('bbl', a.bbl, 'bldgarea',a.bldgarea, 'lotarea', a.lotarea,
+                              'numfloors', a.numfloors, 'blog_lot_ratio', a.bldg_lot_ratio)) as values, 'lotarea_numfloor' as field
+	from (SELECT bbl, bldgarea, lotarea, numfloors, bldgarea::FLOAT/lotarea::FLOAT as bldg_lot_ratio
```

Comment:
When I run this on my machine I get 1914 bbl's that match this criteria, but 1510 (~79%) have numfloors=0. It seems like these records have a separate problem that should go in their own report and not be lumped in with the lotarea/numfloor report. Curious what @AmandaDoyle thinks about this too

Reply:
Yea, I agree with you. We should set an extra condition that numfloors doesn't equal to 0.